src/resources/views/gov-uk-login.blade.php

```diff
@@ -0,0 +1,20 @@
+@section('heading')
+    <x-govuk::notification-banner title="Cookies" colour="blue">
+        <p class="govuk-body">We use some essential cookies to make this service work.</p>
+        <p class="govuk-body">Your browser session will be stored once you sign in.</p>
+    </x-govuk::notification-banner>
+@endsection
+
+@section('after-main')

```

Comment:
Needs to be configurable via config.

Reply:
you mean sections ?

Reply:
The content in the password area.
Don't forget, this is a public library. Anyone could use this.

Reply:
fixed